docs/Modders/Mod-file-structure/4-Unit-related-JSON-files.md

```diff
@@ -6,31 +6,31 @@
-| promotions | List of Strings | defaults to none | A list of all the promotions the unit automatically receives upon being built. Each promotion must be in [UnitPromotions.json](#unitpromotionsjson) |
-| uniques | List of Strings | defaults to none | A list of the unique abilities this unit has. A list of almost all uniques can be found [here](../Unique-parameters.md#unit-uniques) |
-| replacementTextForUniques | String | defaults to none | If provided, this will be displayed instead of the list of uniques. Can be used for better formatting. |
-| attackSound | String | defaults to none | The sound that is to be played when this unit attacks. For possible values, see [Sounds](#../Modders/Images-and-Audio.md#sounds)
-| civilopediaText | List | Default empty | see [civilopediaText chapter](5-Miscellaneous-JSON-files.md#civilopedia-text) |
-
+Each unit has the following structure:
+
+| Attribute | Type | Default | Notes |
+| --------- | ---- | ------- | ----- |
+| name | String | Required | |
+| unitType | String | Required | The type of the unit. Must be in [UnitTypes.json](#unittypesjson) |
+| cost | Integer | -1 | The amount of production required to build this unit. The production needed is always positive |
+| movement | Integer | 0 | The amount of movement points the unit has by |
+| strength | Integer | 0 | The melee attack and defensive strength of the unit. If this and rangedStrength are ommited or 0, the unit will be a civilian |
+| rangedStrength | Integer | 0 | The ranged attack strength of the unit. If omitted, the unit cannot ranged attack. If used, strength must be set too. |
+| religiousStrength | Integer | 0 | The religious attack and defensive strength of the unit |
+| range | Integer | 2 | The range from which ranged attacks can be preformed |
+| interceptRange | Integer | 0 | Air units attacking within in this range will be intercepted |
+| requiredTech | String | none | The tech required to build this unit. Must be in [Techs.json](2-Civilization-related-JSON-files.md#techsjson) |
+| obsoleteTech | String | none | After researching this tech, the unit can no longer be build. Must be in [Techs.json](2-Civilization-related-JSON-files.md#techsjson) |
+| requiredResource | String | none | Resource that is consumed by building this unit. Must be in [TileResources.json](3-Map-related-JSON-files.md#tileresourcesjson) |
+| upgradesTo | String | none | Unit that this unit can upgrade to when it is available. Must be in [Units.json](#unitsjson) |
+| replaces | String | none | If this unit is unique to a nation, this is the unit it replaces. Must be in [Units.json](#unitsjson) |
+| uniqueTo | String | none | The nation that this unit is unique to. Must be in [Nations.json](2-Civilization-related-JSON-files.md#nationsjson) |
+| hurryCostModifier | Integer | 0 | If this unit is bought for gold/faith, it's price is increased by so much percent |
```

Comment:
I believe this only matters for gold, not faith, but someone can correct me on that